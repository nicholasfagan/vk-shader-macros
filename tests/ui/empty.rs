
use vk_shader_macros::include_glsl;

#[allow(dead_code)]
const SPIR_V: &'static [u32] = include_glsl!();

fn main() {}
