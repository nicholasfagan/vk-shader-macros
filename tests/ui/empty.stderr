error: expected path to shader source as literal string
See `https://docs.rs/vk-shader-macros/0.2.6' for help
 --> $DIR/empty.rs:5:32
  |
5 | const SPIR_V: &'static [u32] = include_glsl!();
  |                                ^^^^^^^^^^^^^^^
  |
  = note: this error originates in a macro (in Nightly builds, run with -Z macro-backtrace for more info)
